sorc/ocnice_prep.fd/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# This is the CMake build file for the chgres_cube utility in the
+# UFS_UTILS package.
+#
+# George Gayno, Mark Potts, Kyle Gerheiser
+
+set(lib_src
+    arrays_mod.F90
+    init_mod.F90
+    ocncalc_mod.F90
+    restarts_mod.F90
+    utils_mod.F90
+    utils_esmf_mod.F90)
+
+set(exe_src ocniceprep.F90)
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -r8 -assume byterecl")
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-0 -fdefault-real-8")
+endif()
+
+set(exe_name oiprep)
+
+add_library(ocnice_prep_lib STATIC ${lib_src})
+add_executable(${exe_name} ${exe_src})
+
+set(mod_dir "${CMAKE_CURRENT_BINARY_DIR}/mod")
+set_target_properties(ocnice_prep_lib PROPERTIES Fortran_MODULE_DIRECTORY ${mod_dir})
+target_include_directories(ocnice_prep_lib INTERFACE ${mod_dir})
+
+target_link_libraries(
+  ocnice_prep_lib
+  PUBLIC
+  esmf
+  MPI::MPI_Fortran
+  NetCDF::NetCDF_Fortran)
+
```

Comment:
Is this program threaded? I am able to build without the OpenMP library.

Reply:
No, this code not threaded. Should I remove this?

Reply:
Yes.

Reply:
The OpenMP library, not the MPI library. I would think that MPI is needed because of ESMF.

Reply:
You know, I thought about that! But I see now the comment was supposed to be for the next line and I didn't expand the comment. Let me fix it.